private-templates-service/client/src/components/Dashboard/PreviewModal/PreviewModal.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { ModalBackdrop, ModalWrapper, ACPanel, ACWrapper, DescriptorWrapper } from './styled';
```

Comment:
Is this styled div importing pattern common/used elsewhere? I haven't seen it before